app/inbound/total/get-total-by-calculation-id.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../data')
+
+const getTotalByCalculationId = async (calculationId, transaction) => {
+  return db.total.findOne({
+    transaction,
+    lock: true,
+    where: {
+      calculationId
+    }
+  })
+}
+
+module.exports = getTotalByCalculationId
```

Comment:
This is outside the scope of the ticket.

Reply:
That shouldn't be in here and wasn't showing on my version of the PR as expected, I have checked the files and pushed again, only 5 files are showing as changed as expected if you can confirm please.